src/node/element/path/data.rs

```diff
@@ -327,25 +346,26 @@ mod tests {
         test!("S42,0", SmoothCubicCurve, Absolute, &[42.0, 0.0]);
         test!("s \t 42,0", SmoothCubicCurve, Relative, &[42.0, 0.0]);
 
-        test!("A2.6,0 -7", EllipticalArc, Absolute, &[2.6, 0.0, -7.0]);
-        test!("a 2.6 ,0 -7", EllipticalArc, Relative, &[2.6, 0.0, -7.0]);
-
+        test!("A1 1 2.6,0 0 0 -7", EllipticalArc, Absolute, &[1.0, 1.0, 2.6, 0.0, 0.0, 0.0, -7.0]);
+        test!("a1 1 2.6,0 0 0 -7", EllipticalArc, Relative, &[1.0, 1.0, 2.6, 0.0, 0.0, 0.0, -7.0]);
+        test!("a32 32 0 00.03-45.22", EllipticalArc, Relative, &[32.0, 32.0, 0.0, 0.0, 0.0, 0.03, -45.22]);
```

Comment:
Is there a reference for the correctness of this parsing result?

Reply:
I've created a [CodePen here](https://codepen.io/mlwilkerson/pen/GRjWeLg) that runs these scenarios, along with a few others. It shows a visual representation of the original (with space delimiters around the arc flags), with a modified version laid over the top with reduced opacity and alternate color (to visually see that there's no difference). It also uses DOM APIs to verify that the resulting Path objects have equal length in both cases, and also samples all along each path to verify that the resulting (x, y) coordinate at each sample point is equal in both cases.

So this could be run in any browser to inspect whether that browser's parsing works according to these expectations. I've run it successfully in Chrome, Firefox, and Safari on Mac OS.

(I'm also going to add a couple more test cases, which have also passed in the linked CodePen and are already passing in this library, with the changes proposed in this PR.)

Reply:
It was surprising that “00.03” was interpreted as 0, 0, and 0.03. Does it mean that `0.03` would also result in 0 and 0.03?

Reply:
I think I get it now. These flags are always on position 4 and 5, and they are always either 0 or 1. Hence, we have this interesting parsing sequence, which would indeed be surprising in other contexts.